pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -105,6 +105,7 @@ stdenv.mkDerivation rec {
       ))
       (lib.withFeature ncursesSupport "ncursesw")
       (lib.withFeature systemdSupport "systemd")
+      (lib.withFeatureAs systemdSupport "sysusersdir" "${placeholder "out"}/lib/sysusers.d")
       (lib.withFeatureAs systemdSupport "systemdsystemunitdir" "${placeholder "bin"}/lib/systemd/system/")
```

Comment:
Maybe move sysusersdir so it's next to tmpfilesdir, since they seem to be more similar to each other (same output) than either are to systemdsystemunitdir?